recipe-server/normandy/recipes/api/views.py

```diff
@@ -138,6 +160,32 @@ class RecipeRevisionViewSet(viewsets.ReadOnlyModelViewSet):
     ]
 
 
+class ApprovalRequestViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = ApprovalRequest.objects.all()
+    serializer_class = ApprovalRequestSerializer
+    permission_classes = [
+        AdminEnabledOrReadOnly,
+    ]
+
+    @detail_route(methods=['POST'])
+    def approve(self, request, pk=None):
```

Comment:
Should this raise an error if the approval is already approved or rejected? Same with the reject method below.